.github/workflows/ci.yml

```diff
@@ -21,4 +21,5 @@ jobs:
       with:
         ruby-version: ${{ matrix.ruby-version }}
         bundler-cache: true # 'ruby/setup-ruby' action provides built-in caching
-    - run: bundle exec rspec
+    - name: Run tests
```

Comment:
I recommend to interpolate the matrix element used into the name to make the github actions UI list the matrix specific in list views.